tests/foreman/ui/test_docker.py

```diff
@@ -1399,9 +1399,11 @@ def test_positive_create_with_external_registry(self):
         :CaseLevel: Integration
         """
         repo_name = 'rhel'
-        container_name = gen_string('alphanumeric')
+        container_name = gen_string('alphanumeric').lower()
```

Comment:
Sorry, forgot to mention that. There is a change for UI that makes docker containers names looking more nicely -- it splits names with a space by capital letter and search fails. With lowercase there are no changes in the name.
![scr](https://user-images.githubusercontent.com/2845814/29126896-e0954bb8-7d1f-11e7-8f1a-f6a1f97e9762.png)